package.json

```diff
@@ -10,7 +10,7 @@
   },
   "types": "./types.d.ts",
   "engines": {
-    "node": "^4 || ^5 || ^6 || ^7",
+    "node": "^4 || ^5 || ^6 || ^7 || ^8",
```

Comment:
Those known projects do not use the open source version of MAGE, so I would really rather stick to non end-of-life versions of Node (and not have to support those anymore). If those projects wanna use this version of MAGE, they can in the same breath bump the version of Node they're on, right?